docs/core/install/includes/linux-install-50-yum.md

```diff
@@ -0,0 +1,22 @@
+
+### Install the SDK
+
+.NET SDK allows you to develop apps with .NET. If you install .NET SDK, you don't need to install the corresponding runtime. To install .NET SDK, run the following commands:
+
+```bash
+sudo yum install dotnet-sdk-5.0
+```
+
+### Install the runtime
+
+ASP.NET Core Runtime allows you to run apps that were made with .NET and .NET Core that didn't provide the runtime. The commands below install ASP.NET Core Runtime, which is the most compatible runtime for .NET. In your terminal, run the following commands:
```

Comment:
```suggestion
ASP.NET Core Runtime allows you to run apps that were made with .NET and .NET Core that didn't provide the runtime. The commands below install ASP.NET Core Runtime, which is the most compatible runtime for .NET. In your terminal, run the following command:
```

Reply:
Another example of capitalization inconsistency

>### Install the runtime
>
>ASP.NET Core Runtime